mzLib/MzML/Mzml.cs

```diff
@@ -353,7 +356,8 @@ private static MsDataScan GetMsDataOneBasedScanFromConnection(Generated.mzMLType
             }
 
             if (!msOrder.HasValue || !isCentroid.HasValue)
-                throw new MzLibException("!msOrder.HasValue || !isCentroid.HasValue");
+                return null;
+                //throw new MzLibException("!msOrder.HasValue || !isCentroid.HasValue");
```

Comment:
Is there a way to pass a warning message? Perhaps in an `out` variable?

Reply:
i would like to do something like that. be good to report a list of scans that were ignored. i'm not sure how.

Reply:
https://github.com/smith-chem-wisc/ProteoformSuite/blob/e94716ce9b76395a3ebc1b6a0864090d3e3f3865/ProteoformSuiteInternal/Sweet.cs#L106

Reply:
Something like this? Can use `out` string variables to pass warnings back from method